### PR TITLE
Add Sentry .NET SDK to the registry.

### PR DESCRIPTION
**Pull request by jamescrosswell**

### CODE REVIEW

**Comment by svrnm** [comment]
please sign the CLA.

On the PR: We are currently moving vendor specific distributions over to https://opentelemetry.io/ecosystem/distributions/, I would recommend you raise a different PR that deletes all sentry entries in the registry and adds them to that page.

**Comment by jamescrosswell** [comment]
> please sign the CLA.
> 
> On the PR: We are currently moving vendor specific distributions over to https://opentelemetry.io/ecosystem/distributions/, I would recommend you raise a different PR that deletes all sentry entries in the registry and adds them to that page.

OK. I'm a contractor for Sentry so I'll need to get @bitsandfoxes to track down the CLA manager for Sentry. 

Thanks for the heads up re restructuring the docs. 👍🏻

**Comment by svrnm** [comment]
@jamescrosswell can we close this PR or do you want to have this entry still added to the registry? If so we need you to sign the CLA first

**Comment by jamescrosswell** [comment]
> @jamescrosswell can we close this PR or do you want to have this entry still added to the registry? If so we need you to sign the CLA first

Hi @svrnm , I just signed the CLA... so yes, if we could please merge that would be great!

File: data/registry/tools-dotnet-sentry.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - sentry
+  - error monitoring
+license: MIT
+description:
+  The Sentry OpenTelemetry .NET integration provides a processor and propagator to send data 
+  to Sentry and associate traces spans with Sentry errors. It also includes experimental 
+  support to forward OpenTelemetry metrics to Sentry.   
+  
+  For configuration details, see [OpenTelemetry
+  Support](https://docs.sentry.io/platforms/dotnet/performance/instrumentation/opentelemetry/).
```
Comment (svrnm) [inline data/registry/tools-dotnet-sentry.yml:20]:
```suggestion
```

File: data/registry/tools-dotnet-sentry.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - f#
+  - .net
+  - processor
+  - propagator
+  - utilities
+  - sentry
+  - error monitoring
+license: MIT
+description:
+  The Sentry OpenTelemetry .NET integration provides a processor and propagator to send data
```
Comment (svrnm) [inline data/registry/tools-dotnet-sentry.yml:15]:
```suggestion
  The Sentry .NET integration for OpenTelemetry provides a processor and propagator to send data 
```

See https://www.linuxfoundation.org/legal/trademark-usage

**Review by svrnm — DISMISSED** [review]
@jamescrosswell We can merge this in after the changes have been applied. Note that we need Sentry to take a look at this and migrate their registry entries over to the distribution page eventually.

**Comment by jamescrosswell** [comment]
> @jamescrosswell We can merge this in after the changes have been applied. Note that we need Sentry to take a look at this and migrate their registry entries over to the distribution page eventually.

Understood. @bitsandfoxes do you want to tap someone on the shoulder in the office about this?

**Review by bruno-garcia — APPROVED** [review]

**Review by cartermp — APPROVED** [review]
Thanks for your patience in getting this one in!

**Pull request merged**